lib/logflare/endpoints.ex

```diff
@@ -279,12 +280,20 @@ defmodule Logflare.Endpoints do
     iex> run_query_string(%User{...}, {:bq_sql, "select current_time() where @value > 4"}, params: %{"value" => "123"})
     {:ok, %{rows:  [...]} }
   """
-  @typep run_query_string_opts :: [sandboxable: boolean(), params: map(), parsed_labels: map()]
+  @typep run_query_string_opts ::
+           {:sandboxable, boolean()}
+           | {:parsed_labels, map()}
+           | {:params, map()}
+           | run_query_options()
   @typep language :: :bq_sql | :pg_sql | :lql
   @spec run_query_string(User.t(), {language(), String.t()}, run_query_string_opts()) ::
           run_query_return()
-  def run_query_string(user, {language, query_string}, opts \\ %{}) do
-    opts = Enum.into(opts, %{sandboxable: false, parsed_labels: %{}, params: %{}})
+  def run_query_string(user, {language, query_string}, opts \\ []) do
+    opts =
+      [sandboxable: false, use_query_cache: true, parsed_labels: %{}, params: %{}]
+      |> Keyword.merge(opts)
```

Comment:
`opts` was declared as a keyword list in `@run_query_string_opts` but used a map as the default in the function. I've switched to a keyword list for consistency.